game.rb

```diff
@@ -0,0 +1,50 @@
+require_relative 'environment'
```

Comment:
I am curious, was not needed to add the full route including the file extension (.rb)?

Reply:
- Ruby is gonna try adding some extension until the file is found, if the file at the end is not found a LoadError will be raised.

- It is not needed to add the full route since Ruby is going to try to load the library named string relative to the requiring file's path, basically behind scenes is gonna happen something like this `require(File.expand_path('environment', File.dirname(__FILE__)))`

You can refer to the docs for more info https://ruby-doc.org/core-2.5.0/Kernel.html#method-i-require_relative